embrio-nrf51/src/lib.rs

```diff
@@ -28,19 +28,15 @@ pub struct EmbrioNrf51<'b> {
 }
 
 impl<'b> EmbrioNrf51<'b> {
-    pub fn new(
-        cortex_m: &'b mut cortex_m::Peripherals,
-        nrf51: &'b mut nrf51::Peripherals,
-    ) -> EmbrioNrf51<'b> {
+    pub fn new(nrf51: &'b mut nrf51::Peripherals) -> EmbrioNrf51<'b> {
         let pins = Pins::new(&mut nrf51.GPIO);
-        let uart = Uart::new(&mut nrf51.UART0, &mut cortex_m.NVIC);
+        let uart = Uart::new(&mut nrf51.UART0);
 
         EmbrioNrf51 { pins, uart }
     }
 
     pub fn take() -> Option<EmbrioNrf51<'static>> {
         struct StaticPeripherals {
-            cortex_m: cortex_m::Peripherals,
```

Comment:
Wasn't 100% sure about removing this because there presumably could be reason to reintroduce it in the future, though it's currently an unused field and also not accessible via public api so...